### PR TITLE
feat:  Mobile views for Ongoing and Filled TX

### PR DESCRIPTION
**Pull request by Tulun**
Mobile view for TX History tables.

Screen Shot 2022-03-22 at 2 04 40 PM

### CODE REVIEW

**Review by mrice32 — APPROVED** [review]
LGTM!

File: src/views/Transactions/useTransactionsView.ts [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
+
   return {
     provider,
     chainId,
     isConnected,
     account,
     initOnboard: init,
     transactions,
+    // windowSize can return undefined -- default to 0 for easier typing.
+    width: width || 0,
```
Comment (daywiss) [inline src/views/Transactions/useTransactionsView.ts:24]: are we gauranteed to always get a resize event on start?

**Review by daywiss — APPROVED** [review]
nice work, and thanks for refactoring the main table mapper, lgtm

File: src/views/Transactions/Transactions.tsx [hunk 2]
```diff
@@ -30,6 +46,24 @@ const Transactions = () => {
...
+    [transactions, setOpenFilledRow]
+  );
+
+  const mobileOngoingTx = useMemo(
+    () =>
+      createMobileTransactionTableJSX(
+        transactions.filter((x) => x.filled < 100),
+        setOpenOngoingRow
+      ),
+    [transactions, setOpenOngoingRow]
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/views/Transactions/Transactions.tsx:64]: We might be able to do a check on the width?

**Review by Gamaranto — APPROVED** [review]

**Pull request merged**